cryptostore/data/parquet.py

```diff
@@ -32,6 +32,15 @@ def __init__(self, exchanges, config=None):
         self.del_file = True
         self.file_name = config.get('file_format') if config else None
         self.path = config.get('path') if config else None
+        if config and 'compression' in config:
+            self.comp_codec = config['compression']['codec']
+            if 'level' in config['compression']:
+                self.comp_level = config['compression']['level']
+            else:
```

Comment:
instead of having these else clauses, cant you just set the two vars to None before the if statement? It would be easier to read this way (in my opinion)

Reply:
@bmoscon 
You are fully right! Cleaning done.